foundations-macros/src/settings.rs

```diff
@@ -883,6 +913,26 @@ mod tests {
             }
         };
 
+        #[cfg(feature = "settings-deny-unknown-fields")]
+        let expected = add_deny_unknown_fields_attr(expected);
+
         assert_eq!(actual, expected);
     }
+
+    #[cfg(feature = "settings-deny-unknown-fields")]
+    fn add_deny_unknown_fields_attr(mut code: String) -> String {
```

Comment:
I would prefer it to be done via the macro rather than code string substitution

Reply:
Not necessary now that there's no feature flag being used :+1: